controlpanel/frontend/jinja2/includes/app-list.html

```diff
@@ -31,12 +31,10 @@
         {{ yes_no(user_is_app_admin) }}
       </td>
       <td class="govuk-table__cell">
-      {% if user_is_app_admin or request.user.is_superuser %}
```

Comment:
Does this mean that all users see the "Manage app" button for all apps?

Reply:
Yes, they can click through to the app details page, but non-admin users will not be able to edit anything. I did this for consistency with other lists of resources.